packages/govern-create/tasks/sequences/deploy-factory.ts

```diff
@@ -0,0 +1,57 @@
+import { task } from 'hardhat/config'
+import { getGovernRegistry, setHRE } from '../../helpers/helpers'
+import {
+  deployGoverBaseFactory,
+  deployGoverFactory,
+  deployGoverQueueFactory,
+  deployGoverTokenFactory,
+} from '../../helpers/delploys'
+import { Address, eContractid } from '../../helpers/types'
+import { registerContractInJsonDb } from '../../helpers/artifactsDb'
+
+const {
+  GovernBase,
+  GovernBaseFactory,
+  GovernFactory,
+  GovernTokenFactory,
+  QueueBase,
+  GovernQueueFactory,
+} = eContractid
+
+task('deploy-factory', 'Deploys an GovernBaseFactory instance')
```

Comment:
Why not cover those tasks with simple and well-named ``Deployer`` object which will increase the readability and lower the code duplications.